widgets/contact/contact.php

```diff
@@ -63,6 +63,12 @@ function get_widget_form(){
 						'description' => __('Where contact emails will be delivered to.', 'so-widgets-bundle'),
 						'sanitize' => 'multiple_emails',
 					),
+					'default_sender' => array(
+						'type' => 'text',
+						'label' => __('Sender email address', 'so-widgets-bundle'),
+						'description' => __('Sender address to be used when there is no email field defined.', 'so-widgets-bundle'),
+						'sanitize' => 'sanitize_email',
```

Comment:
It's possible that a user might still miss that this is a required field if they don't want an 'email' field in their contact form, so we still need some default value in code to bypass validation requirements in this case.